core/controllers/profile.py

```diff
@@ -370,31 +370,40 @@ def put(self):
 
 
 class UserInfoHandler(base.BaseHandler):
-    """Provides info about user."""
+    """Provides info about user.
```

Comment:
Don't add a new line after `user.` just continue with the comment.

Reply:
It would be more than 80 characters

Reply:
Then split it in the middle of the sentence.

Reply:
Done